src/metabase/query_processor/middleware/fetch_source_query.clj

```diff
@@ -94,42 +96,81 @@
         (log/info (trs "Trimming trailing comment from card with id {0}" card-id))
         trimmed-string))))
 
+(defn sub-cached-field-refs
```

Comment:
```suggestion
(defn- sub-cached-field-refs
```

since it's not used anywhere else. I'm worried about this function and I had to search the whole PR to see where else it was used. `defn-` would have saved me some time.